llvm/lib/Target/AIE/AIEInterBlockScheduling.cpp

```diff
@@ -712,10 +712,14 @@ void InterBlockScheduling::enterRegion(MachineBasicBlock *BB,
   auto &BS = getBlockState(BB);
   DEBUG_BLOCKS(dbgs() << "    >> enterRegion, Iter=" << BS.FixPoint.NumIters
                       << "\n");
+
+  // Only add regions of loops when in the GatheringRegions phase
   if (BS.Kind != BlockType::Loop ||
       BS.FixPoint.Stage == SchedulingStage::GatheringRegions) {
-    // Only add regions of loops when in the GatheringRegions phase
-    BS.addRegion(BB, RegionBegin, RegionEnd);
+    const int NumTopFixedInstructions = 0;
```

Comment:
nit: the parameters are of type `size_t` (`unsigned`) and we are using `int` here.  One alterative is:

```
BS.addRegion(BB, RegionBegin, RegionEnd, 0 /*NumTopFixedInstructions*/,
                 0 /*NumBotFixedInstructions*/);
```

Reply:
This is gone now :)